### PR TITLE
Change risk level route name to community level

### PR DESCRIPTION
**Pull request by faihegberg**
Change route name from risk level to community level, as per [card](https://trello.com/c/e7cUfzd1/611-change-explainer-link-to-covid-community-level-metrics).

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -184,8 +191,31 @@ export default function App() {
...
+                      ) {
+                        return (
+                          <Redirect
+                            to={`covid-community-level-metrics${props.location.hash}`}
+                          />
+                        );
+                      }
+                      return <Redirect to="/covid-community-level-metrics" />;
+                    }}
+                  />
```
Comment (faihegberg) [inline src/App.tsx:216]: Let me know if there's a better solution here. At first I tried:
```js
                  <Redirect
                    from="/covid-risk-levels-metrics#icu-capacity-used"
                    to="/covid-community-level-metrics#icu-capacity-used"
                  />
                  <Redirect
                    from="/covid-risk-levels-metrics#daily-new-cases"
                    to="/covid-community-level-metrics#daily-new-cases"
                  />
                  <Redirect
                    from="/covid-risk-levels-metrics#positive-test-rate"
                    to="/covid-community-level-metrics#positive-test-rate"
                  />
                  <Redirect
                    from="/covid-risk-levels-metrics#infection-rate"
                    to="/covid-community-level-metrics#infection-rate"
                  />
```
And the redirect seems to disregard the hash (i.e. it redirects to the correct page but doesn't scroll to the appropriate section). Tried playing around with props like [exact](https://v5.reactrouter.com/web/api/Redirect/exact-bool) and [strict](https://v5.reactrouter.com/web/api/Redirect/strict-bool) too. Found one possible solution in [this thread](https://stackoverflow.com/questions/45864594/react-router-redirect-hash-link).

File: src/App.tsx [hunk 2]
```diff
@@ -57,6 +57,13 @@ const AfterUnsubscribe = lazy(() =>
   import('screens/Subscriptions/AfterUnsubscribe'),
 );
 
+const metricExplainerHashes = [
+  '#icu-capacity-used',
+  '#daily-new-cases',
```
Comment (faihegberg) [inline src/App.tsx:62]: This was taken from what we currently have in the code. I think `daily-new-cases` hash is dead though. We can probably get rid of this and have it redirect to the top of the page.

File: src/App.tsx [hunk 3]
```diff
@@ -311,35 +341,35 @@ export default function App() {
...
                   <Redirect
                     from="/old-hospitalization-projections"
-                    to="/covid-risk-levels-metrics#icu-capacity-used"
+                    to="/covid-community-level-metrics#icu-capacity-used"
                   />
                   <Redirect
                     from="/daily-new-cases-explained"
-                    to="/covid-risk-levels-metrics#daily-new-cases"
+                    to="/covid-community-level-metrics#daily-new-cases"
                   />
```
Comment (faihegberg) [inline src/App.tsx:361]: Same comment as above. I think `#daily-new-cases` hash is dead.

File: src/App.tsx (hunk 1 above)
Comment (mikelehen) [reply src/App.tsx:216]: Hrm.  Is there a reason we have to actually validate that the hash is in `metricExplainerHashes`?  Why not just always preserve the hash with 

```
                          <Redirect
                            to={`covid-community-level-metrics${props.location.hash}`}
                          />
```

or whatever?

File: src/App.tsx (hunk 2 above)
Comment (mikelehen) [reply src/App.tsx:62]: I think it was replaced by `weekly-new-cases` (if you click the "About this data" link under the weekly new cases chart and Learn more, that's where you go), so we should maybe use that instead.

File: src/App.tsx (hunk 3 above)
Comment (mikelehen) [reply src/App.tsx:361]: TBH we could probably remove all 8 of these redirects.  Seems unlikely anybody is linking to them anymore...  Else if we keep them, I'd update "daily-new-cases" to "weekly-new-cases."

**Review by mikelehen — APPROVED** [review]
LGTM with a couple minor comments.

File: src/App.tsx (hunk 1 above)
Comment (faihegberg) [reply src/App.tsx:216]: We could definitely do that. I just put this check there to be explicit about which hashes _should_ be valid ones. That said, the behaviour should be the same either way. I'll keep it simple and stick with your solution.

File: src/App.tsx (hunk 3 above)
Comment (faihegberg) [reply src/App.tsx:361]: Let's get rid of them 👍🏼

**Comment by faihegberg** [comment]
[Change explainer link to /covid-community-level-metrics](https://trello.com/c/e7cUfzd1/611-change-explainer-link-to-covid-community-level-metrics)

**Pull request merged**